Report_templates/qwtimeseries.Rmd

```diff
@@ -33,7 +33,7 @@ isData <- nrow(qw_historic)!=0
 ```{r eval=isData, echo=FALSE, warning=FALSE, message=FALSE, fig.height = 14, fig.width=11}
 depth <- filterParmData(qw_historic, "00098")
 totalP <- filterParmData(qw_historic, "00665", isTotalP = TRUE, depth) 
-chlorophyll <- filterParmData(qw_historic, c("32210", "70953"), depth)
+chlorophyll <- filterParmData(qw_historic, c("32210", "70953"), isChlorophyll = TRUE, depth)
```

Comment:
Might want to switch the order to match the order that the function accepts the arguments since the depth data frame argument is not named (e.g. `depth_df = depth`). Since you named `isChlorophyll` it does work, but could cause confusion later.